samples/spring-petclinic/workload.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020 The Kubernetes Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: spring-petclinic
+spec:
+  ports:
+  - port: 80
+    targetPort: 8080
+  selector:
+    app: spring-petclinic
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: spring-petclinic
+  labels:
+    app: spring-petclinic
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: spring-petclinic
+  template:
+    metadata:
+      labels:
+        app: spring-petclinic
+    spec:
+      containers:
+        - name: workload
+          # built with paketo builder from https://github.com/spring-projects/spring-petclinic
+          image: ghcr.io/vmware-tanzu/servicebinding/test/petclinic:20200922
```

Comment:
Should we push this image to `ghcr.io/servicebinding`? Also looks like the image is from 2020. A new build would be nice.

Reply:
updated the image ref to ghcr.io/servicebinding/runtime/samples/spring-petclinic@sha256:83ab44832a1db6c03d34e758199b0d9cbf29ce5beeaac4fbf96443a63342b3d4

> A new build would be nice.

It's a sample, the age of the image isn't important as long as it works. If anything, showing that an older image still works is a good thing.

Reply:
ok, thanks!